diff --git a/adapter/uhdf2/model/display/buffer/v1_0/hdi_impl/display_buffer_hdi_impl.h b/adapter/uhdf2/model/display/buffer/v1_0/hdi_impl/display_buffer_hdi_impl.h
index f0134de1..4fe2ba7a 100644
--- a/adapter/uhdf2/model/display/buffer/v1_0/hdi_impl/display_buffer_hdi_impl.h
+++ b/adapter/uhdf2/model/display/buffer/v1_0/hdi_impl/display_buffer_hdi_impl.h
@@ -29,7 +29,7 @@ namespace Buffer {
 namespace V1_0 {
 class DisplayBufferHdiImpl : public IDisplayBuffer {
 public:
-    DisplayBufferHdiImpl(bool isAllocLocal = false);
+    explicit DisplayBufferHdiImpl(bool isAllocLocal = false);
     virtual ~DisplayBufferHdiImpl() = default;
 
     virtual int32_t AllocMem(const AllocInfo &info, BufferHandle *&handle) const override;
diff --git a/framework/model/display/driver/adapter_soc/hi35xx_disp.c b/framework/model/display/driver/adapter_soc/hi35xx_disp.c
index 7ab604b4..bd30312e 100644
--- a/framework/model/display/driver/adapter_soc/hi35xx_disp.c
+++ b/framework/model/display/driver/adapter_soc/hi35xx_disp.c
@@ -67,7 +67,7 @@ static void Lcd8BitMuxCfg(unsigned long ioCfgBase)
     OSAL_WRITEL(0x462, ioCfgBase + 0x0060);
 }
 
-void Lcd24BitMuxCfg(unsigned long ioCfgBase)
+static void Lcd24BitMuxCfg(unsigned long ioCfgBase)
 {
     OSAL_WRITEL(0x462, ioCfgBase + 0x0034);
     OSAL_WRITEL(0x432, ioCfgBase + 0x0058);
diff --git a/framework/model/display/driver/hdf_drm_panel.c b/framework/model/display/driver/hdf_drm_panel.c
index eeb93d9b..4e2f5b21 100644
--- a/framework/model/display/driver/hdf_drm_panel.c
+++ b/framework/model/display/driver/hdf_drm_panel.c
@@ -243,8 +243,9 @@ static struct HdfDrmPanel *InstanceHdfDrmPanel(struct DispManager *manager, int3
     return hdfDrmPanel;
 }
 
-int32_t HdfDrmPanelEntryInit(struct HdfDeviceObject *object)
+static int32_t HdfDrmPanelEntryInit(struct HdfDeviceObject *object)
 {
+    (void)object;
     uint32_t ret;
     uint32_t i;
     uint32_t j;
diff --git a/framework/model/display/driver/panel/ili9881_st_5p5.c b/framework/model/display/driver/panel/ili9881_st_5p5.c
index 36393d2c..d6a8f1fb 100755
--- a/framework/model/display/driver/panel/ili9881_st_5p5.c
+++ b/framework/model/display/driver/panel/ili9881_st_5p5.c
@@ -605,7 +605,7 @@ static void PanelResInit(struct panel_ili9881_dev *panel_dev)
     panel_dev->hw_delay.unprepare_delay = 20;  /* 20:unprepare_delay */
 }
 
-int32_t PanelEntryInit(struct HdfDeviceObject *object)
+static int32_t PanelEntryInit(struct HdfDeviceObject *object)
 {
     struct device_node *panelNode = NULL;
     struct panel_ili9881_dev *panel_dev = NULL;
diff --git a/framework/model/display/driver/panel/ili9881c_boe.c b/framework/model/display/driver/panel/ili9881c_boe.c
index b5565029..07c6d209 100644
--- a/framework/model/display/driver/panel/ili9881c_boe.c
+++ b/framework/model/display/driver/panel/ili9881c_boe.c
@@ -414,7 +414,7 @@ static struct DsiCmdDesc g_panelOffCode[] = {
 #define ILI9881C_MIPI_DATA_LANES   4
 
 static int Ili9881cBoeSendCmds(struct mipi_dsi_device *dsi,
-	const struct DsiCmdDesc *cmds, int size)
+    const struct DsiCmdDesc *cmds, int size)
 {
     int32_t i;
 
diff --git a/framework/model/display/driver/panel/mipi_icn9700.c b/framework/model/display/driver/panel/mipi_icn9700.c
index 99c1d49d..279ed825 100644
--- a/framework/model/display/driver/panel/mipi_icn9700.c
+++ b/framework/model/display/driver/panel/mipi_icn9700.c
@@ -289,7 +289,7 @@ static void Icn9700PanelInit(struct PanelData *panel)
     panel->off = Icn9700Off;
 }
 
-int32_t Icn9700EntryInit(struct HdfDeviceObject *object)
+static int32_t Icn9700EntryInit(struct HdfDeviceObject *object)
 {
     struct Icn9700Dev *icn9700 = NULL;
 
diff --git a/framework/model/display/driver/panel/ssp_st7789.c b/framework/model/display/driver/panel/ssp_st7789.c
index 8f39f6cf..67063c8d 100644
--- a/framework/model/display/driver/panel/ssp_st7789.c
+++ b/framework/model/display/driver/panel/ssp_st7789.c
@@ -90,7 +90,7 @@ static int32_t SpiWrite9Bits(DevHandle spiHandle, uint8_t cmd, uint8_t data)
     return HDF_SUCCESS;
 }
 
-DevHandle GetSpiHandle(void)
+static DevHandle GetSpiHandle(void)
 {
     int32_t ret;
     DevHandle spiHandle = NULL;
@@ -259,7 +259,7 @@ static struct PanelData g_panelData = {
     .setBacklight = St7789SetBacklight,
 };
 
-int32_t SspSt7789EntryInit(struct HdfDeviceObject *object)
+static int32_t SspSt7789EntryInit(struct HdfDeviceObject *object)
 {
     if (object == NULL) {
         HDF_LOGE("%s: param is null", __func__);
